CMSgov/mint-app
# [EASI-3505, EASI-4076] updating apollo/client and node v20 🎉

# EASI-3505, EASI-4076



https://jiraent.cms.gov/browse/EASI-3505
## Changes and Description

- updated to node v20.11.1 LTS
- remove apollo since it has been discontinued ([end of life blog post](https://www.apollographql.com/blog/announcing-the-end-of-life-schedule-for-apollo-cli-service-commands))
- updated apollo client to 3.9.6



## How to test this change

1. `yarn install`
2. `scripts/dev down && scripts/dev up`
3. Verify all things are well




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
